int32 mode
int32 motor1_angle
int32 motor2_angle
int32 PF
int32 EV
float32 t
